src/backend/functions/orchestrator/lib/index.js

```diff
@@ -11,23 +11,37 @@ const composeP = (...fns) => R.composeWith(R.then)(fns);
 // snakeToCamel :: String -> String
 const snakeToCamel = R.replace(/([_][a-z])/g, o(R.replace('_', ''), R.toUpper));
 
+// pascalToCamel :: String -> String
+const pascalToCamel = R.replace(/^\w/g, R.toLower);
+
+// keyConverter :: (String -> String) -> {k: v} -> {k: v}
+const keyConverter = f => R.compose(R.fromPairs, R.map(R.adjust(0, f)), R.toPairs);
+
 function noop() {}
 
 // convertEnvVars :: {k: v} -> {k: v}
 const convertEnvVars = R.pipe(
-    R.toPairs,
-    R.map(R.adjust(0, o(snakeToCamel, R.toLower))),
-    R.fromPairs,
+    keyConverter(o(snakeToCamel, R.toLower)),
     R.evolve({subnets: subnets => [...subnets.split(',').map(R.trim)]})
 );
 
-const stopTranscription = R.curry((ecs, {mediaUrl, cluster, task, tasksTableName}) => {
-   return ecs.stopTask({cluster, task})
+// convertImage :: {k: v} -> {k: v}
+const convertImage = keyConverter(pascalToCamel);
+
+// stopTranscription :: AWS.ECS -> {k: v} -> Promise {k: v}
+const stopTranscription = R.curry((ecs, {mediaUrl, cluster, taskArn, tasksTableName}) => {
+   return ecs.stopTask({cluster, task: taskArn})
        .promise()
        .then(() => ({mediaUrl, tasksTableName}));
 });
 
-// startTranscription :: AWS.ECS -> AWS.DDB -> Promise {k: v}
+// createEnrichedError :: {k: v} -> Error -> Error
+const createEnrichedError = R.curry((params, err) => {
+   console.log('Error: ' + err);
```

Comment:
should this be `console.error`?

Reply:
I treat lambdas like a Docker and put every to `stdout` but I'm probably going to move to an actual logger like Bunyan or Winston.

Reply:
fair enough good call on something like winston